### PR TITLE
lvm2: version bumped to 2.03.04.

### PR DESCRIPTION
**Pull request by Florin65**

### CODE REVIEW

**Comment by lunar-linux-b01** [comment]
SUCCESS!

File: filesys/lvm2/BUILD [hunk 1]
```diff
@@ -1,24 +1,47 @@
...
 fi
 
+OPTS+=" --with-udev_prefix=/usr"
 if module_installed systemd; then
-  OPTS+=" --with-udev_prefix=/usr \
-          --with-systemdsystemunitdir=$(pkg-config systemd --variable=systemdsystemunitdir)"
+  OPTS+=" --with-systemdsystemunitdir=/usr/lib/systemd/system"
+fi
+
+if ( module_installed udev ) || ( module_installed eudev ); then
```
Comment (dagbrown) [inline filesys/lvm2/BUILD]: You don't need parens here.  The parens just cause excess forks to happen.

    if module_installed udev || module_installed eudev; then

would be better.

**Comment by Florin65** [comment]
retest this please

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Pull request merged**